src/main/java/org/jabref/gui/actions/ActionHelper.java

```diff
@@ -13,4 +18,22 @@ public static BooleanExpression needsDatabase(StateManager stateManager) {
     public static BooleanExpression needsEntriesSelected(StateManager stateManager) {
         return Bindings.isNotEmpty(stateManager.getSelectedEntries());
     }
+
+    public static BooleanExpression needsEntriesSelected(int numberOfEntries, StateManager stateManager) {
+        return Bindings.createBooleanBinding(
+                () -> stateManager.getSelectedEntries().size() == numberOfEntries,
+                stateManager.getSelectedEntries());
+    }
+
+    public static BooleanExpression isFieldSetForSelectedEntry(Field field, StateManager stateManager) {
+        return isAnyFieldSetForSelectedEntry(Collections.singletonList(field), stateManager);
+    }
+
+    public static BooleanExpression isAnyFieldSetForSelectedEntry(List<Field> fields, StateManager stateManager) {
+        BibEntry entry = stateManager.getSelectedEntries().get(0);
+        return Bindings.createBooleanBinding(
+                () -> !Collections.disjoint(fields, entry.getFields()),
```

Comment:
I think `entry.getFields().stream().anyMatch(fields::contains)` is slightly better performance wise (as `disjoint` needs to check if *all* elements of `fields` are not contained in `entry.getFields()`)